src/analysis/ast_iterators.ml

```diff
@@ -45,7 +129,12 @@ let build_uid_to_locs_tbl ~(local_defs : Mtyper.typedtree) () =
   uid_to_locs_tbl
 
 let iter_on_usages ~f (local_defs : Mtyper.typedtree) =
-  let iter = Cmt_format.iter_on_occurrences ~f in
+  let occ_iter = Cmt_format.iter_on_occurrences ~f in
```

Comment:
Does this need any change to handle cms files?

Reply:
No. `iter_on_occurrences` does not contain cmt-specific logic. In fact, `Cms_format` uses `Cmt_format.iter_on_occurrences`.